README.md

```diff
@@ -6,6 +6,12 @@ A general repository for AI / GenAI packages and explorations.
 
 | Package | Description | Issues | Version |
 | --- | --- | --- | --- |
+| [dart_mcp](pkgs/dart_mcp/) | Library for making MCP severs and clients | ![issues][dart_mcp_issues] | [![pub package](https://img.shields.io/pub/v/dart_mcp.svg)](https://pub.dev/packages/dart_mcp) |
```

Comment:
If a package is published (or is something we intend to publish), I'd take the text here verbatim from the pubspec description.

perhaps ==> `A package for making MCP servers and clients.`